proto/cacheclient.proto

```diff
@@ -48,3 +51,39 @@ message _SetResponse {
   ECacheResult result = 1;
   string message = 2;
 }
+
+message _HashGetRequest {
+  bytes cache_hash_name = 1;
+  bytes cache_hash_key = 2;
```

Comment:
Maybe just hash_name and hash_key? They're going to be handled quite differently so maybe it would be confusing to overlap the names.

Reply:
Changed to "dict_name" and "dict_key"